api/helper/process-logger-results.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * @module api/helper/process-logger-results
+ */
+
+/**
+ * Takes in a survey_instance model and processes it into a format recognized by app.
+ * @param {Object} logs - list of objects of survey result logs
+ * @returns {Object} processed list of logs in an array
+ */
+function processLoggerResults (logs) {
+    return {
+        eventName: logs.eventName,
+        metaData: JSON.stringify(logs.metaData),
```

Comment:
Right, but the metaData field in the database is a string, which means it is either a `CHAR` or a `VARCHAR` both have limits.